### PR TITLE
feat(ecs/eip): add auto_pay support

### LINKED ISSUE #2034: [ECS/EIP]Add autopay support for prepaid resources
I'd like to add autopay argument support for users want to pay prepaid order with discounts and coupons.

### PR DESCRIPTION
**Pull request by niuzhenguo**
**What this PR does / why we need it**:

**Which issue this PR fixes**:
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
fixes #2034 

**Special notes for your reviewer**:
No tests add for this as it need to pay the order on Console.

**Release note**:

```release-note

```

## PR Checklist

* [x] Tests added/passed.
* [x] Documentation updated.
* [x] Schema updated.

## Acceptance Steps Performed

```
make testacc TEST=./huaweicloud/ TESTARGS='-run=TestAccComputeV2Instance'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud/ -v -run=TestAccComputeV2Instance -timeout 360m -parallel 4
=== RUN   TestAccComputeV2Instance_basic
=== PAUSE TestAccComputeV2Instance_basic
=== RUN   TestAccComputeV2Instance_disks
=== PAUSE TestAccComputeV2Instance_disks
=== RUN   TestAccComputeV2Instance_prePaid
=== PAUSE TestAccComputeV2Instance_prePaid
=== RUN   TestAccComputeV2Instance_tags
=== PAUSE TestAccComputeV2Instance_tags
=== RUN   TestAccComputeV2Instance_powerAction
=== PAUSE TestAccComputeV2Instance_powerAction
=== CONT  TestAccComputeV2Instance_basic
=== CONT  TestAccComputeV2Instance_tags
=== CONT  TestAccComputeV2Instance_prePaid
=== CONT  TestAccComputeV2Instance_powerAction
--- PASS: TestAccComputeV2Instance_basic (224.99s)
=== CONT  TestAccComputeV2Instance_disks
--- PASS: TestAccComputeV2Instance_prePaid (246.79s)
--- PASS: TestAccComputeV2Instance_disks (229.88s)

--- PASS: TestAccComputeV2Instance_tags (471.99s)
--- PASS: TestAccComputeV2Instance_powerAction (693.63s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud       693.667s
```
```
make testacc TEST=./huaweicloud/services/acceptance/eip/ TESTARGS='-run=TestAccEIPAssociate'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud/services/acceptance/eip/ -v -run=TestAccEIPAssociate -timeout 360m -parallel 4
=== RUN   TestAccEIPAssociate_basic
=== PAUSE TestAccEIPAssociate_basic
=== RUN   TestAccEIPAssociate_compatible
=== PAUSE TestAccEIPAssociate_compatible
=== CONT  TestAccEIPAssociate_basic
=== CONT  TestAccEIPAssociate_compatible
--- PASS: TestAccEIPAssociate_compatible (115.01s)
--- PASS: TestAccEIPAssociate_basic (121.18s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/eip       121.221s
```

### CODE REVIEW

File: huaweicloud/common/resource_schema.go [hunk 1]
```diff
@@ -82,6 +82,20 @@ func SchemaAutoRenew(conflicts []string) *schema.Schema {
 	return &resourceSchema
 }
 
+func SchemaAutoPay(conflicts []string) *schema.Schema {
+	resourceSchema := schema.Schema{
+		Type:     schema.TypeString,
```
Comment (ShiChangkuo) [inline huaweicloud/common/resource_schema.go]: how about using **TypeBool**?

File: huaweicloud/services/eip/resource_huaweicloud_vpc_eip.go [hunk 2]
```diff
@@ -232,6 +233,9 @@ func resourceVpcEIPV1Create(d *schema.ResourceData, meta interface{}) error {
 			IsAutoPay:   "true",
 			IsAutoRenew: d.Get("auto_renew").(string),
 		}
+		if d.Get("auto_pay").(string) == "false" {
+			chargeInfo.IsAutoPay = "false"
+		}
```
Comment (ShiChangkuo) [inline huaweicloud/services/eip/resource_huaweicloud_vpc_eip.go]: suggest to merge it with L233

File: huaweicloud/common/resource_schema.go (hunk 1 above)
Comment (niuzhenguo) [reply huaweicloud/common/resource_schema.go]: done

File: huaweicloud/services/eip/resource_huaweicloud_vpc_eip.go (hunk 2 above)
Comment (niuzhenguo) [reply huaweicloud/services/eip/resource_huaweicloud_vpc_eip.go]: done

**Review by ShiChangkuo — APPROVED** [review]

**Pull request merged**